mangaki/mangaki/utils/data.py

```diff
@@ -71,7 +71,7 @@ def load_csv(self, filename, convert=float, title_filename=None):
         triplets = np.array(triplets, dtype=np.object)
         self.anonymized = AnonymizedData(
             X=triplets[:, 0:2],
-            y=triplets[:, 2],
+            y=np.array(triplets[:, 2], dtype=np.float32),
```

Comment:
Let's use `np.float64` rather than `np.float32`, memory is cheap and numerical errors are hard to debug (that is, precision is not cheap.)